pending-documentation.md

```diff
@@ -276,16 +277,193 @@ syntax Pgm ::= underbar(NameWithUnderbar)
 
 ### `unused` attribute
 
-K will warn you if you declare a symbol that is not used in any of the rules of your
-definition. Sometimes this is intentional, however; in this case, you can suppress the
-warning by adding the `unused` attribute to the production or cell.
+K will warn you if you declare a symbol that is not used in any of the rules of
+your definition. Sometimes this is intentional, however; in this case, you can
+suppress the warning by adding the `unused` attribute to the production or
+cell.
 
 ```
 syntax Foo ::= foo() [unused]
 
 configuration <foo unused=""> .K </foo>
 ```
 
+### Symbol priority and associativity
+
+Unlike most other parser generators, K combines the task of parsing with AST
+generation. A production declared with the `syntax` keyword in K is both a
+piece of syntax used when parsing, and a symbol that is used when rewriting.
+As a result, it is generally convenient to describe expression grammars using
+priority and associativity declarations rather than explicitly transforming
+your grammar into a series of nonterminals, one for each level of operator
+precedence. Thus, for example, a simple grammar for addition and multiplication
+will look like this:
+
+```k
+syntax Exp ::= Exp "*" Exp
+             | Exp "+" Exp
+```
+
+However, this grammar is ambiguous. The term `x+y*z` might refer to `x+(y*z)`
+or to `(x+y)*z`. In order to differentiate this, we introduce a partial
+ordering between productions known as priority. A symbol "has tighter priority"
+than another symbol if the first symbol can appear under the second, but the
+second cannot appear under the first without a bracket. For example, in 
+traditional arithmetic, multiplication has tighter priority than addition,
+which means that `x+y*z` cannot parse as `(x+y)*z` because the addition
+operator would appear directly beneath the multiplication, which is forbidden
+by the priority filter.
+
+Priority is applied individually to each possible ambiguous parse of a term. It
+then either accepts or rejects that parse. If there is only a single remaining
+parse (after all the other disambiguation steps have happened), this is the
+parse that is chosen. If all the parses were rejected, it is a parse error. If
+multiple parses remain, they might be resolved by further disambiguation such
+as via the `prefer` and `avoid` attributes, but if multiple parses remain after
+disambiguation finishes, this is an ambiguous parse error, indicating there is
+not a unique parse for that  term. In this case, you can get all possible
+parses that were valid by means of the `--gen-glr-parser` command line flag.
```

Comment:
I would remove this sentence about `--gen-glr-parser` flag. It makes it sound like the purpose of this flag is for returning all ambiguous parses, which is not the purpose of that flag. Instead say that K will print out the ambiguous parses and the user must fix it by adding brackets where needed or changing their grammar.